cmd/vorteil/main.go

```diff
@@ -121,6 +122,38 @@ const (
 	sourceINVALID            = "INVALID"
 )
 
+func readSourcePath(src string) (string, string, error) {
+
+	src, err := filepath.Abs(src)
+	if err != nil {
+		return "", "", err
+	}
+
+	// Check if source contains a target
+	colonSplit := strings.Split(src, ":")
+	var target string
+
+	// colonSplit will always be 1 when strings.splitting something that doesn't have it
+	if len(colonSplit) > 1 {
+		src = colonSplit[0]
+		target = colonSplit[1]
+
+		// Windows has one extra coloumn due to C:\ from abs
+		if runtime.GOOS == "windows" {
+			src = fmt.Sprintf("%s%s", "C:", colonSplit[1])
```

Comment:
what happens if I have disk D:?

Reply:
I ended up using the colonSplit variable properly to replace it with whatever letter gets split